metrics/scripts/averages.py

```diff
@@ -16,7 +16,9 @@ def geomean(n):
     # geometric_mean was not added to statistics until Python 3.8
     if (sys.version_info.major > 3) or (
             sys.version_info.major == 3 and sys.version_info.minor >= 8):
-        return statistics.geometric_mean(n)
+        if 0 in n:
+            return 0
+        return geometric_mean(n)
```

Comment:
Previously this branch was being used to prevent `geometric_mean` from being imported with Python versions that didn't exist.  I don't think this new code will work with older Python versions.  That may be okay, but I'm not sure what @tomdeakin plans for compatibility here.